app/serializers/park_show_serializer.rb

```diff
@@ -1,3 +1,5 @@
 class ParkShowSerializer < ActiveModel::Serializer
   attributes :id, :name, :city, :state, :zip, :rating, :description, :photo
+
+  has_many :reviews
 end
```

Comment:
these two serializers look like they do nearly the exact same thing. We should be merging them into one serializer